plugins/qtAccessionPlugin/lib/model/QubitAccessionEventI18n.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+
+/**
+ * Skeleton subclass for representing a row from the 'accession_event_i18n' table.
+ *
+ * 
+ *
```

Comment:
Same issues than in QubitAccessionEvent.php.

Reply:
I think you copied the comments from the auto-generated table map class. Check the other custom model classes, as we add the other comment at the beginning.